Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -309,6 +309,29 @@ QString GameFile::GetLanguage(DiscIO::Language lang) const
   }
 }
 
+QString GameFile::GetFileType() const
+{
+  switch (GetBlobType())
+  {
+  case DiscIO::BlobType::PLAIN:
+    return QObject::tr("Plain");
```

Comment:
This makes sense codewise but I think it might confuse end-users, especially for something tha'll show on the UI. Maybe using "ISO" on this case?